packages/marshal/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./jsconfig.json",
```

Comment:
why keep both files? I'm pretty sure `tsconfig` alone can do the job these are in concert.

Reply:
tsconfig is for generating the `.d.ts` file. I don't want to generate those and clutter the workspace when type checking, and simply forcing `noEmit true` on the command line for `lint:types` didn't work (conflicting value with `emitDeclarationOnly`)

Reply:
Happy to rename to `jsconfig.build.json` if you believe that's a better name.

Reply:
> Happy to rename to `jsconfig.build.json` if you believe that's a better name.

I like this better, assuming you mean: rename `tsconfig.json` to `jsconfig.build.json`.

Reply:
Yes, I think if this config file is to be used only for building then it would help to have that conveyed in the filename.

Separately (and not now), I'd like to consider using `tsconfig` instead of `jsconfig` so that `yarn tsc` to typecheck doesn't require an option. My understanding is that with the right options a tsconfig can work the same as a jsconfig.

Reply:
Done